ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/Shared/_ProjectHeader.cshtml

```diff
@@ -12,6 +12,7 @@
     </div>
     @if (!string.IsNullOrEmpty(Context.Request.Query["rd"]))
     {
+     
```

Comment:
unnecessary whitespace

Reply:
Either I've addressed this already or its due to the way github displays things.